package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcr",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Would prefer to bump the version myself as the next version will be a semver-major.